scripts/coverage.sh

```diff
@@ -1,136 +1,66 @@
 #!/usr/bin/env bash
 #
-# Runs all tests and collects code coverage
+# Run tests and collect code coverage
 #
-# Warning: this process is a little slow
+# Run all:
+#   $ ./script/coverage.sh
 #
+# Run for specific packages
+#   $ ./script/coverage.sh -p solana-account-decoder
+#   $ ./script/coverage.sh -p solana-account-decoder -p solana-accounts-db [-p ...]
 
-if ! command -v grcov; then
-  echo "Error: grcov not found.  Try |cargo install grcov|"
-  exit 1
-fi
-
-if [[ ! "$(grcov --version)" =~ 0.8.[0-9] ]]; then
-  echo Error: Required grcov version not installed
+set -e
+here=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 
-  echo "Installed version: $(grcov --version)"
+# Check for grcov commands
+if ! command -v grcov >/dev/null 2>&1; then
+  echo "Error: grcov not found.  Try |cargo install grcov|"
   exit 1
 fi
 
-set -e
-cd "$(dirname "$0")/.."
-source ci/_
+# shellcheck source=ci/rust-version.sh
+source "$here/../ci/rust-version.sh" nightly
```

Comment:
considering `-C instrument-coverage` is also available for the stable channel, i think we need explicit comment for sticking with nightly still.

how about like this?:

```
Use nightly as we have some nightly-only tests (frozen-abi)
```

Or, better yet, add `+nightly test_abi_` inside the check-1 ci step...? so that frozen abi tests failure can be detected faster and we can use stable rustc. cc: @apfitzge

Reply:
fixed: [c9fe41c](https://github.com/anza-xyz/agave/pull/1260/commits/c9fe41c8b2d13720b2c879c7c93bb21788820498)